org.unknown_horizons.UnknownHorizons.json

```diff
@@ -0,0 +1,219 @@
+{
+    "app-id": "org.unknown_horizons.UnknownHorizons",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "20.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "unknown-horizons",
+    "rename-desktop-file": "unknown-horizons.desktop",
+    "rename-appdata-file":  "unknown-horizons.appdata.xml",
+    "rename-icon": "unknown-horizons",
+    "finish-args": [
+        "--share=ipc",
```

Comment:
```suggestion
```
you have that with fallback-x11 :)

Reply:
I looked at this after recent comment by @mwleeds asking if I'm really sure and – not really, I can't find lines that would imply it's true.

Reply:
Ah, my bad. I assumed it's after reading that somewhere on a review here :) thanks for the confirmation that it's not really a thing